docs/existing_environment.md

```diff
@@ -0,0 +1,68 @@
+# Working with the an existing environment
+
+If you just want to contribute to this codebase, and test your changes against an existing environment, follow these
+steps. The configuration values shown here will be for the Rabble Rouser core team's [staging environment](https://demo.rr-staging.click).
+If you're going to work with a different environment, you'll need to change the values accordingly.
+
+If you're trying to create a brand new environment, follow [these instructions](./new_environment.md) instead.
+
+## Dependencies
+
+Make sure you [install them](../README.md#dependencies) first.
+
+## Set up AWS
+1. Ask an existing team member to create an AWS user for you
+2. [Configure an API key pair for the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html)
+
```

Comment:
Are we sure that we want to make our AWS environment available for anyone who asks?